### PR TITLE
test flow accumulation with new assertions

### PR DESCRIPTION
**Pull request by itati01**
The pull request contains new assertions for the flow accumulation as discussed in #168 . It also does not rely on external efficiency grids. I was surprised to see that the maximum flow accumulation = outlet cell appears twice in the accumulation arrays. However, this facilitates the testing, especially for Dinf. A deprecation warning of `pytest` is also removed.

### CODE REVIEW

File: tests/test_grid.py [hunk 1]
```diff
@@ -277,7 +315,8 @@ def test_to_ascii():
     catch = d.catch
     fdir = d.fdir
     grid.clip_to(catch)
-    grid.to_ascii(fdir, 'test_dir.asc', target_view=fdir.viewfinder, dtype=np.float)
+    # np.float is depreciated
+    grid.to_ascii(fdir, 'test_dir.asc', target_view=fdir.viewfinder, dtype=float)
```
Comment (mdbartos) [inline tests/test_grid.py:319]: Can you actually change this to np.float64?

File: tests/test_grid.py [hunk 2]
```diff
@@ -11,8 +11,8 @@
 dir_path = os.path.join(data_dir, 'dir.asc')
 dem_path = os.path.join(data_dir, 'dem.tif')
 roi_path = os.path.join(data_dir, 'roi.tif')
-eff_path = os.path.join(data_dir, 'eff.tif')
-dinf_eff_path = os.path.join(data_dir, 'dinf_eff.tif')
+# eff_path = os.path.join(data_dir, 'eff.tif')
```
Comment (mdbartos) [inline tests/test_grid.py:14]: Can you delete these commented-out lines? The old lines will still be in the history if needed.

File: tests/test_grid.py [hunk 3]
```diff
@@ -38,15 +38,15 @@ class Datasets():
 fdir = grid.read_ascii(dir_path, dtype=np.uint8, crs=grid.crs)
 dem = grid.read_raster(dem_path)
 roi = grid.read_raster(roi_path)
-eff = grid.read_raster(eff_path)
-dinf_eff = grid.read_raster(dinf_eff_path)
+# eff = grid.read_raster(eff_path)
```
Comment (mdbartos) [inline tests/test_grid.py:41]: Same goes for these commented out lines. And below as well.

**Comment by mdbartos** [comment]
Posted a couple quick comments. Could you change these lines? Thx.

**Comment by itati01** [comment]
Sure. I have removed some more comments referring to the efficiency grids. Additionally, the grid files are removed.

**Review by mdbartos — APPROVED** [review]

**Comment by mdbartos** [comment]
Looks good to me!

**Pull request merged**